docs/getting-started/ios/fastlane-swift.md

```diff
@@ -0,0 +1,39 @@
+# Getting Started with Fastlane.swift
+
+Welcome to Fastlane.swift. Fastlane.swift allows you to write your _fastlane_ configuration using Xcode, in Swift - the language you have come to know and love from the world of iOS development..
+
+## Currently Supported:
+
+Fastlane.swift currently supports all actions in the main, [_fastlane_](https://github.com/fastlane/fastlane) repository. The API for each action is automatically generated and will accept all options and parameters.
```

Comment:
Extra `,`.

```
The API for each action is automatically generated and will accept all options and parameters.
```
As a user I'd expect that, so it might be a sentence we can just remove?

Reply:
👍